addon/services/user-activity.js

```diff
@@ -38,6 +40,7 @@ export default Service.extend(Evented, {
       this.get('scrollActivity').on('scroll', this, this._handleScroll);
     } else if (this._eventsListened.indexOf(eventName) === -1) {
       this._eventsListened.push(eventName);
+      if (this.get('_isFastBoot')) { return; }
```

Comment:
Before push seems fine, since the `scrollActivity` service has its own code to handle fastboot compat